internal/endtoend/testdata/process_plugin_sqlc_gen_json/exec.json

```diff
@@ -1,3 +1,4 @@
 {
+  "process": "sqlc-gen-json",
```

Comment:
I didn't have sqlc-gen-json in my path so I added this to get the tests to pass I can remove it.